policyengine_us/parameters/gov/states/nc/tax/income/deductions/itemized/cap/real_estate.yaml

```diff
@@ -0,0 +1,25 @@
+description: North Carolina limits the real estate tax deduction to the following amount, based on filing status.
+
+metadata:
+  lable: North Caroliona real estate tax cap
+  period: year
+  unit: currency-USD
+  reference:
+    - title: North Carolina Department of Revenue
+      href: https://www.ncdor.gov/taxes-forms/individual-income-tax/north-carolina-standard-deduction-or-north-carolina-itemized-deductions 
+    # This cap is not mentioned in the legal docuemnt, as it is part of the salt limitation
```

Comment:
I don't follow - what subsection is this in? If it's based on the federal salt cap then we need to point to that parameter rather than duplicating it.